src/python/pants/util/strutil.py

```diff
@@ -144,7 +144,7 @@ def strip_v2_chroot_path(v: bytes | str) -> str:
     return re.sub(r"/.*/process-execution[a-zA-Z0-9]+/", "", v)
 
 
-def hard_wrap(s: str, *, indent: int = 0, width: int = 96) -> Sequence[str]:
+def hard_wrap(s: str, *, indent: int = 0, width: int = 96) -> list[str]:
```

Comment:
I don't see any reason to not be specific in what kind of data is returned, and is needed in order to work with the data effectively besides iterating over it..